style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: large;
+}
+
+#main {
+  background-color: #f5f5f5;
+  width: 100%;
+  text-align: center;
+  padding: 1.5rem;
+}
+
+#title {
+  font-size: 4rem;
+  margin-bottom: 0;
```

Comment:
I think we don't need margin as default value of margin is already zero

Reply:
- [ ] Your comment here is very nice and you told to the student the reason why he should do the request changes :+1: . Good job :smile: